### PR TITLE
Feature 630/member profile card updates closes#630

### PR DESCRIPTION
**Pull request by zsbrown97**

### CODE REVIEW

**Review by mkimberlin — APPROVED** [review]

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx [hunk 1]
```diff
@@ -32,50 +37,61 @@ const MemberSummaryCard = ({ member, index }) => {
...
+                className={"large"}
+                src={getAvatarURL(workEmail)}
+              />
+            }
+          ></CardHeader>
+        </Container>
+          <CardContent>
+            <Container fixed className={"info-container"}>
+              <Typography variant="body2" color="textSecondary" component="p">
+                <a href="mailto:williamsh@objectcomputing.com">
```
Comment (meyerinj) [inline web-ui/src/components/member-directory/MemberSummaryCard.jsx:62]: Holly is going to get a lot of emails.

File: web-ui/src/context/AppContext.js [hunk 2]
```diff
@@ -244,7 +244,7 @@ const AppContextProvider = (props) => {
     let currentMonth = new Date().getMonth() + 1;
     let newDate = prevCheckinDate ? new Date(...prevCheckinDate) : new Date();
     if (prevCheckinDate) {
-      let prevCheckinMonth = prevCheckinDate.getMonth() + 1;
```
Comment (meyerinj) [inline web-ui/src/context/AppContext.js:247]: Is this a stray change? Seems unconnected to the member profile card.

**Review by meyerinj — CHANGES_REQUESTED** [review]

File: web-ui/src/context/AppContext.js (hunk 2 above)
Comment (mkimberlin) [reply web-ui/src/context/AppContext.js:247]: it's actually preventing a really strange error. I can't explain it yet...but this fixes it.

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx (hunk 1 above)
Comment (hwilliams314) [reply web-ui/src/components/member-directory/MemberSummaryCard.jsx:62]: yay

Comment (hwilliams314) [reply web-ui/src/components/member-directory/MemberSummaryCard.jsx:62]: @zsbrown97

**Review by mkimberlin — APPROVED** [review]

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx [hunk 3]
```diff
@@ -32,50 +37,61 @@ const MemberSummaryCard = ({ member, index }) => {
...
+                className={"large"}
+                src={getAvatarURL(workEmail)}
+              />
+            }
+          ></CardHeader>
+        </Container>
+          <CardContent>
+            <Container fixed className={"info-container"}>
+              <Typography variant="body2" color="textSecondary" component="p">
+                <a href={workEmail}>
```
Comment (mkimberlin) [inline web-ui/src/components/member-directory/MemberSummaryCard.jsx:62]: Actually...this should be
```
<a href={`mailto:${workEmail}`}>
```

Comment (zsbrown97) [reply web-ui/src/components/member-directory/MemberSummaryCard.jsx]: Done

**Review by meyerinj — APPROVED** [review]

**Pull request merged**